api/model.go

```diff
@@ -22,28 +22,29 @@ import (
 type Volume struct {
 	client.Resource
 
-	Name                    string                 `json:"name"`
-	Size                    string                 `json:"size"`
-	Frontend                types.VolumeFrontend   `json:"frontend"`
-	DisableFrontend         bool                   `json:"disableFrontend"`
-	FromBackup              string                 `json:"fromBackup"`
-	NumberOfReplicas        int                    `json:"numberOfReplicas"`
-	DataLocality            types.DataLocality     `json:"dataLocality"`
-	StaleReplicaTimeout     int                    `json:"staleReplicaTimeout"`
-	State                   types.VolumeState      `json:"state"`
-	Robustness              types.VolumeRobustness `json:"robustness"`
-	EngineImage             string                 `json:"engineImage"`
-	CurrentImage            string                 `json:"currentImage"`
-	BackingImage            string                 `json:"backingImage"`
-	Created                 string                 `json:"created"`
-	LastBackup              string                 `json:"lastBackup"`
-	LastBackupAt            string                 `json:"lastBackupAt"`
-	LastAttachedBy          string                 `json:"lastAttachedBy"`
-	Standby                 bool                   `json:"standby"`
-	RestoreRequired         bool                   `json:"restoreRequired"`
-	RevisionCounterDisabled bool                   `json:"revisionCounterDisabled"`
-	DiskSelector            []string               `json:"diskSelector"`
-	NodeSelector            []string               `json:"nodeSelector"`
+	Name                    string                   `json:"name"`
+	Size                    string                   `json:"size"`
+	Frontend                types.VolumeFrontend     `json:"frontend"`
+	DisableFrontend         bool                     `json:"disableFrontend"`
+	FromBackup              string                   `json:"fromBackup"`
+	NumberOfReplicas        int                      `json:"numberOfReplicas"`
+	ReplicaAutoBalance      types.ReplicaAutoBalance `json:"replicaAutoBalance"`
+	DataLocality            types.DataLocality       `json:"dataLocality"`
+	StaleReplicaTimeout     int                      `json:"staleReplicaTimeout"`
```

Comment:
nit: put a space around these to keep the replica stuff together in their own section.

Reply:
Thanks, addressed.